src/libraries/System.Runtime.InteropServices/tests/ComInterfaceGenerator.Tests/Generated/Microsoft.Interop.ComInterfaceGenerator/Microsoft.Interop.ComInterfaceGenerator/ComInterfaceGenerator.Tests.IDerivedComInterface.cs

```diff
@@ -9,6 +9,36 @@
 [System.Runtime.InteropServices.DynamicInterfaceCastableImplementationAttribute]
 file unsafe partial interface InterfaceImplementation : global::ComInterfaceGenerator.Tests.IDerivedComInterface
 {
+    [System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.Interop.ComInterfaceGenerator", "42.42.42.42")]
+    [System.Runtime.CompilerServices.SkipLocalsInitAttribute]
+    int global::ComInterfaceGenerator.Tests.IComInterface1.GetData()
```

Comment:
The runtime will never execute these methods that are explicit implementations of the base interface method on the implementation type, so we can stub these out to just throw `new UnreachableException()` if it's easier.

Reply:
Also since this is an interface type, we could also just not implement them and things would work.

Reply:
> Also since this is an interface type, we could also just not implement them and things would work.

I agree in principle but I'd prefer to have an explicit exception we control rather than the slightly confusing - `MethodNotFoundException` or what that would be thrown. We can provide a nice message, in the future or now, that tells people they are doing something wrong and get them back on track.